x/bank/keeper/msg_server.go

```diff
@@ -29,20 +29,14 @@ func (k msgServer) Send(goCtx context.Context, msg *types.MsgSend) (*types.MsgSe
 	if err := k.SendEnabledCoins(ctx, msg.Amount...); err != nil {
 		return nil, err
 	}
-	from, err := sdk.AccAddressFromBech32(msg.FromAddress)
-	if err != nil {
-		return nil, err
-	}
-	to, err := sdk.AccAddressFromBech32(msg.ToAddress)
-	if err != nil {
-		return nil, err
-	}
+	from := sdk.AccAddress(msg.FromAddress)
```

Comment:
How does it work exist prefix validation by bech32?('link...')

Reply:
Client should assign binary address value to `Address` type of all messages.
We will have to think about this and discuss it.

Reply:
Ah, I temporally wrong things thought that the address bytes contained bech32 prefix bytes. I understood now.